### ISSUE TITLE
All records blank. UnboundLocalError: local variable 'amount' referenced before assignment

### ISSUE THREAD

**Issue by profmugshot** [max-mykhailenko/memTask#9]
Using ST3 on Windows. Installed via Package manager, restarted program, maybe system too.
Worked on files for hours, records still blank.
```
Traceback (most recent call last):
  File "E:\Program Files\Sublime Text 3\sublime_plugin.py", line 543, in run_
    return self.run(edit, **args)
  File "memTask in C:\Users\Max\AppData\Roaming\Sublime Text 3\Installed Packages\memTask.sublime-package", line 221, in run
  File "memTask in C:\Users\Max\AppData\Roaming\Sublime Text 3\Installed Packages\memTask.sublime-package", line 253, in printLine
UnboundLocalError: local variable 'amount' referenced before assignment

```

**Comment by max-mykhailenko** [comment]
@profmugshot Try it after update and sublime relaunch.

**Comment by vali98n** [comment]
I have the bug too.
Using: ST3, Windows 8.1 x64.

**Comment by max-mykhailenko** [comment]
Possible fixed in 0.2.1, feel free to reopen if you will have problems
